packages/python-pynvim/build.sh

```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://github.com/neovim/pynvim
 TERMUX_PKG_DESCRIPTION="Python client for Neovim"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=0.5.0
+TERMUX_PKG_VERSION=0.5.2
 TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=$TERMUX_PKG_HOMEPAGE/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=448414e8d005b6d99868c8badeec7a20b10a7a37fb6b85fb12846b80c044c279
+TERMUX_PKG_SHA256=734a2432db8683519f58572617528ecb4a2f321bc7b27f034b3f9b2322c15615
```

Comment:
You can also enable auto-updates for this package to have it handled by our auto-update system in the future.
```suggestion
TERMUX_PKG_SHA256=734a2432db8683519f58572617528ecb4a2f321bc7b27f034b3f9b2322c15615
TERMUX_PKG_AUTO_UPDATE=true
```